lib/Service/StackService.php

```diff
@@ -181,6 +181,7 @@ public function findAllArchived($boardId) {
 				if (array_key_exists($card->id, $labels)) {
 					$cards[$cardIndex]->setLabels($labels[$card->id]);
 				}
+				$cards[$cardIndex]->setAttachmentCount($this->attachmentService->count($card->getId()));
```

Comment:
Looks good, though I guess it will be good update tests.
https://github.com/nextcloud/deck/blob/master/tests/unit/Service/StackServiceTest.php#L123